.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Fixes #...
+
+Related to #...
+
+
+#### Changes
+* ...
+* ...
+
+
+#### Checklist (Delete irrelevant sub-headings)
+##### Adding New Features
+* [ ] Have I manually tested my new feature?
+* [ ] Is my new feature sufficiently well tested?
+* [ ] Have I updated the project documentation and README to reflect my changes?
+* [ ] Does the new feature conform to the desired project architecure?
+
+##### Updating Existing Features
+* [ ] Have I manually tested my changes?
```

Comment:
Can we have checkboxes phrased as statements rather than questions?

It seems weird to check a question.